guides/manual-installation.md

```diff
@@ -587,6 +587,143 @@ docker run -d \
+
+##### Enabling SAML when using RPM packages
+
+Provide the following environment variables to trento-web configuration, which is stored at `/etc/trento/trento-web` and restart the application to enable SAML integration.
+
+```
+# Required:
+ENABLE_SAML=true
+SAML_IDP_ID=<<SAML_IDP_ID>>
+SAML_SP_ID=<<SAML_SP_ID>>
+# One of SAML_METADATA_URL or SAML_METADATA_CONTENT
+SAML_METADATA_URL=<<SAML_METADATA_URL>>
+SAML_METADATA_CONTENT=<<SAML_METADATA_CONTENT>>
+
+# Optional:
+SAML_IDP_NAMEID_FORMAT=<<SAML_IDP_NAMEID_FORMAT>>
+SAML_SP_DIR=<<SAML_SP_DIR>>
+SAML_SP_ENTITY_ID=<<SAML_SP_ENTITY_ID>>
+SAML_SP_CONTACT_NAME=<<SAML_SP_CONTACT_NAME>>
+SAML_SP_CONTACT_EMAIL=<<SAML_SP_CONTACT_EMAIL>>
+SAML_SP_ORG_NAME=<<SAML_SP_ORG_NAME>>
+SAML_SP_ORG_DISPLAYNAME=<<SAML_SP_ORG_DISPLAYNAME>>
+SAML_SP_ORG_URL=<<SAML_SP_ORG_URL>>
+SAML_USERNAME_ATTR_NAME=<<SAML_USERNAME_ATTR_NAME>>
+SAML_EMAIL_ATTR_NAME=<<SAML_EMAIL_ATTR_NAME>>
+SAML_FIRSTNAME_ATTR_NAME=<<SAML_FIRSTNAME_ATTR_NAME>>
+SAML_LASTNAME_ATTR_NAME=<<SAML_LASTNAME_ATTR_NAME>>
+SAML_SIGN_REQUESTS=<<SAML_SIGN_REQUESTS>>
+SAML_SIGN_METADATA=<<SAML_SIGN_METADATA>>
+SAML_SIGNED_ASSERTION=<<SAML_SIGNED_ASSERTION>>
+SAML_SIGNED_ENVELOPES=<<SAML_SIGNED_ENVELOPES>>
+```
+
+##### Enabling SAML when using Docker images
+
+Provide the following environment variables to the docker container and restart the application to enable SAML integration.
+
+```bash
+docker run -d \
+-p 4000:4000 \
+--name trento-web \
+--network trento-net \
+--add-host "host.docker.internal:host-gateway" \
+
+...[other settings]...
+
+-e ENABLE_SAML=true
+-e SAML_IDP_ID=<<SAML_IDP_ID>> \
+-e SAML_SP_ID=<<SAML_SP_ID>> \
+# One of SAML_METADATA_URL or SAML_METADATA_CONTENT
+-e SAML_METADATA_URL=<<SAML_METADATA_URL>> \
+-e SAML_METADATA_CONTENT=<<SAML_METADATA_CONTENT>> \
+
+# Optional:
+-e SAML_IDP_NAMEID_FORMAT=<<SAML_IDP_NAMEID_FORMAT>> \
+-e SAML_SP_DIR=<<SAML_SP_DIR>> \
+-e SAML_SP_ENTITY_ID=<<SAML_SP_ENTITY_ID>> \
+-e SAML_SP_CONTACT_NAME=<<SAML_SP_CONTACT_NAME>> \
+-e SAML_SP_CONTACT_EMAIL=<<SAML_SP_CONTACT_EMAIL>> \
+-e SAML_SP_ORG_NAME=<<SAML_SP_ORG_NAME>> \
+-e SAML_SP_ORG_DISPLAYNAME=<<SAML_SP_ORG_DISPLAYNAME>> \
+-e SAML_SP_ORG_URL=<<SAML_SP_ORG_URL>> \
+-e SAML_USERNAME_ATTR_NAME=<<SAML_USERNAME_ATTR_NAME>> \
+-e SAML_EMAIL_ATTR_NAME=<<SAML_EMAIL_ATTR_NAME>> \
+-e SAML_FIRSTNAME_ATTR_NAME=<<SAML_FIRSTNAME_ATTR_NAME>> \
+-e SAML_LASTNAME_ATTR_NAME=<<SAML_LASTNAME_ATTR_NAME>> \
+-e SAML_SIGN_REQUESTS=<<SAML_SIGN_REQUESTS>> \
+-e SAML_SIGN_METADATA=<<SAML_SIGN_METADATA>> \
+-e SAML_SIGNED_ASSERTION=<<SAML_SIGNED_ASSERTION>> \
+-e SAML_SIGNED_ENVELOPES=<<SAML_SIGNED_ENVELOPES>> \
+
+...[other settings]...
+```
+
+### SAML usage variables
+
+Find in this table the SAML integration options and their default values:
+
+| Name                           | Description                                                                                                      | Default                                     |
```

Comment:
Table formatting is broken https://github.com/trento-project/docs/blob/d97a3796850ca77f366d3d5f0fe1440657ff7152/guides/manual-installation.md#saml-usage-variables